odoo_module_migrate/migration_scripts/migrate_120_130.py

```diff
@@ -25,9 +25,15 @@
         r"\.sudo\((?P<user>[^/)]+?)\)": r".with_user(\g<user>)",
         r"\.suspend_security": ".sudo",
         r"\"base_suspend_security\",\n": "",
+        r"self.env.user.company_id": r"self.env.company"
```

Comment:
Can be done in a separate PR

Reply:
In two separate PRs? It wouldn't make much sense to pass one without the other. The multi-company style requires both changes. This is so unless you have not understood something

Reply:
I mean that this part of the change is easier to check and merge.

Basically, all migration scripts have to be applied at the same time, but that doesn't mean that we can add all the scripts in a single PR.
I understand when we have two changes in py and xml and one doesn't work without another. But it seems not the case. Both changes are about multi-company, but they don't depend on each other